lib/runners/processor/lizard.rb

```diff
@@ -0,0 +1,69 @@
+require "etc"
+
+module Runners
+  class Processor::Lizard < Processor
+    include Python
+
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base
+
+      let :issue, object(
+        NLOC: integer,
+        CCN: integer,
+        token: integer,
+        PARAM: integer,
+        length: integer,
+        function: string,
+        long_name: string,
+        )
+    end
+
+    register_config_schema(name: :lizard, schema: Schema.runner_config)
+
+    def analyze(changes)
+      delete_unchanged_files(changes)
+
+      capture3!(analyzer_bin,
+        '--working_threads', Etc.nprocessors.to_s,
+        '--verbose',
+        '--csv',
+        '-o', report_file,
```

Comment:
[suggestion] How about using the long name so that newcomers can understand it easily?

```suggestion
        '--output_file', report_file,
```